scripts/generate-vectors.ts

```diff
@@ -153,7 +161,7 @@ async function getRandomVector(
  * @returns The random vectors.
  */
 async function getRandomVectors(
-  spec: 'bip32' | 'slip10',
+  spec: 'bip32' | 'slip10' | 'cip3',
```

Comment:
I'd be explicit that we are supporting here the "Icarus" derivation described in CIP-3, because CIP-3 documents actually various derivation schemes and it may not be obvious we are referring to "Icarus" only

```suggestion
  spec: 'bip32' | 'slip10' | 'cip3-icarus',
```